test/dubbo/dubbo2.7/pom.xml

```diff
@@ -0,0 +1,38 @@
+<!--
```

Comment:
The path `test/dubbo/dubbo2.7` is incorrect. It should be `test/dubbo/dubbo-2.7`.

Also, this test is failing for me when I run it on my local machine with jdk1.8.

Reply:
```
SpecialAgent.linkRule("dubbo-2.7"[7], sun.misc.Launcher$AppClassLoader@18b4aac2): compatible = null, RulePath: /var/folders/8p/vwxskfsj1nb9s1wwv_98r_r80000gn/T/opentracing-specialagent2403118498880484519/META-INF/plugins/dubbo-2.7-1.6.1-SNAPSHOT.jar
ClassFingerprint#compatible[true]("org.apache.dubbo.rpc.Result")
ClassFingerprint#compatible[true]("org.apache.dubbo.rpc.RpcException")
ClassFingerprint#compatible[true]("org.apache.dubbo.rpc.Invoker")
ClassFingerprint#compatible[true]("org.apache.dubbo.rpc.Filter")
ClassFingerprint#compatible[true]("org.apache.dubbo.rpc.Invocation")
ClassFingerprint#compatible[true]("org.apache.dubbo.rpc.support.RpcUtils")
ClassFingerprint#compatible[true]("org.apache.dubbo.rpc.RpcContext")
ClassFingerprint#compatible[true]("org.apache.dubbo.common.Node")
Allowing instrumentation with "dubbo-2.7" due to "fingerprint.bin match" for:
file:/var/folders/8p/vwxskfsj1nb9s1wwv_98r_r80000gn/T/opentracing-specialagent2403118498880484519/META-INF/plugins/dubbo-2.7-1.6.1-SNAPSHOT.jar
[dubbo-2.7] Target class loader is: sun.misc.Launcher$AppClassLoader@18b4aac2 (system)
[dubbo-2.7] Injection of instrumentation classes deferred
-------> Intercept [DubboAgentRule@main]: public java.util.List org.apache.dubbo.common.extension.ExtensionLoader.getActivateExtension(org.apache.dubbo.common.URL,java.lang.String,java.lang.String)
-------> Intercept [DubboAgentRule@main]: public java.util.List org.apache.dubbo.common.extension.ExtensionLoader.getActivateExtension(org.apache.dubbo.common.URL,java.lang.String,java.lang.String)
-------> Intercept [DubboAgentRule@main]: public java.util.List org.apache.dubbo.common.extension.ExtensionLoader.getActivateExtension(org.apache.dubbo.common.URL,java.lang.String,java.lang.String)
Spans: [{traceId:1, spanId:3, parentId:2, operationName:"GreeterService/sayHello"}, {traceId:1, spanId:2, parentId:0, operationName:"GreeterService/sayHello"}]
Span: {traceId:1, spanId:3, parentId:2, operationName:"GreeterService/sayHello"}
	Component: java-dubbo
	Tags: {component=java-dubbo, span.kind=server, remoteAddress=30.5.121.27:64023}
	Logs:
Span: {traceId:1, spanId:2, parentId:0, operationName:"GreeterService/sayHello"}
	Component: java-dubbo
	Tags: {component=java-dubbo, span.kind=client, remoteAddress=30.5.121.27:15000}
	Logs:
[INFO]
[INFO] --- maven-install-plugin:2.4:install (default-install) @ dubbo-2.7 ---
[INFO] Installing /Users/limfriend/WorkSpace/code/tools/specialagent/test/dubbo/dubbo-2.7/target/dubbo-2.7-1.6.1-SNAPSHOT.jar to /Users/limfriend/.m2/repository/io/opentracing/contrib/specialagent/test/dubbo/dubbo-2.7/1.6.1-SNAPSHOT/dubbo-2.7-1.6.1-SNAPSHOT.jar
[INFO] Installing /Users/limfriend/WorkSpace/code/tools/specialagent/test/dubbo/dubbo-2.7/pom.xml to /Users/limfriend/.m2/repository/io/opentracing/contrib/specialagent/test/dubbo/dubbo-2.7/1.6.1-SNAPSHOT/dubbo-2.7-1.6.1-SNAPSHOT.pom
[INFO] Installing /Users/limfriend/WorkSpace/code/tools/specialagent/test/dubbo/dubbo-2.7/target/dubbo-2.7-1.6.1-SNAPSHOT-tests.jar to /Users/limfriend/.m2/repository/io/opentracing/contrib/specialagent/test/dubbo/dubbo-2.7/1.6.1-SNAPSHOT/dubbo-2.7-1.6.1-SNAPSHOT-tests.jar
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 20.853 s
```